src/node/rpc/node_frontend.h

```diff
@@ -354,6 +354,30 @@ namespace ccf
       make_read_only_endpoint("primary", HTTP_HEAD, is_primary)
         .set_forwarding_required(ForwardingRequired::Never)
         .install();
+
+      auto consensus_config = [this](CommandEndpointContext& args) {
+        // Query node for configurations, separate current from pending
+        if (consensus != nullptr)
+        {
+          auto cfg = consensus->get_latest_configuration();
```

Comment:
Isn't this awkward during configuration changes when there are more than one configuration active at a time? I think we should print all configurations since we pop unused ones on global commit. The command will still return one configuration in most cases.

Reply:
Yeah right now this is only used after a global commit gate in the tests, but in general we should have the historical version too, I agree.